src/tests/static-files/compose.yaml.golden

```diff
@@ -0,0 +1,18 @@
+name: test
+services:
+  x:
+    image: blah
+    networks:
+      default: null
+    x-defang-static-files: ./folder
+  "y":
```

Comment:
the double quotes are to prevent interpretation to a boolean?

Reply:
I think so; the loader does that